definitions/npm/platform_v1.x.x/flow_>=v0.21.x/platform_v1.x.x.js

```diff
@@ -0,0 +1,22 @@
+declare module 'platform' {
+  declare class Platform extends Object {
```

Comment:
It is supposed to be an object, but defining the class makes more sense for the parse method which returns a new instance.